src/structures/clip_game_round.ts

```diff
@@ -0,0 +1,81 @@
+import { getMajorityCount } from "../helpers/discord_utils";
+import ClipAction from "../enums/clip_action";
+import GameRound from "./game_round";
+import type QueriedSong from "./queried_song";
+
+export default class ClipGameRound extends GameRound {
+    /** The Discord Guild ID */
+    public readonly guildID: string;
```

Comment:
Should we just add this to `Round`?

Reply:
I limited it here since it's not used in Round, but I can move it for future use